Views/createblogpost.php

```diff
@@ -24,8 +24,8 @@
         <option value='#Music'>Music</option>
         <option value='#Entertainment'>Entertainment</option>
         <option value='#MindandBody'>Mind and Body</option>
-        <option value='FoodandDrink'>Food and Drink</option>
-        <option value='Fitness'>Fitness</option>
+        <option value='#FoodandDrink'>Food and Drink</option>
```

Comment:
Line 27 and 28 originally had no hashtag in front of the word. Checking to see if it made difference.